packages/keycloak-user-management/src/constants.tsx

```diff
@@ -12,6 +12,9 @@ export const KEYCLOAK_URL_USERS = '/users';
 export const KEYCLOAK_URL_RESET_PASSWORD = '/reset-password';
 export const KEYCLOAK_URL_REQUIRED_USER_ACTIONS = '/authentication/required-actions/';
 
+// OpenSRP API strings
+export const OPENSRP_CREATE_PRACTITIONER_ENDPOINT = 'practitioner/user';
```

Comment:
Should we have a forward slash at the beginning i.e '/practioner/user`

Reply:
i suggest the slash should be at the end not in the beginging .. because a slash at the end is easy to handle as compared the slash at the begining. 

For example if we have api call of of "**xyz.com/**" as **BASEURL** and a end point of **"thing/"** as **ENDPOINT** with **ID "123"**

then in that case if we make a request to it be easier to code them like **_BASEURL+ENDPOINT+ID_** i.e **xyz.com/thing/123**
but now if we consider this same case with slash in the begining we have to prepend / with id everytime we do a request so then the code will be _**BASEURL+ENDPOINT+"/"+ID**_

Reply:
@kelvin-muchiri the base url already has a slash at the end

Reply:
@ashfahan @KipSigei noted